components/layout/ContactNavigation.js

```diff
@@ -0,0 +1,37 @@
+import classes from "./AboutUsNavigation.module.css";
```

Comment:
I would recommend using the same name for JSX module and CSS module (e.g. `ContactNavigation.module.css`).